src/overlays/actors/ovl_Obj_Mure2/z_obj_mure2.h

```diff
@@ -10,7 +10,10 @@ typedef void (*ObjMure2ActionFunc)(struct ObjMure2*, PlayState*);
 typedef struct ObjMure2 {
     /* 0x000 */ Actor actor;
     /* 0x144 */ ObjMure2ActionFunc actionFunc;
-    /* 0x148 */ char unk_148[0x38];
+    /* 0x148 */ Actor* actors[12];
```

Comment:
```suggestion
    /* 0x148 */ Actor* actors[OBJ_MURE2_CHILD_MAX];
```

Think this should be defined as a constant and used appropriately; the 12 in sChildCounts shouldn't be replaced with this constant since number of bushes should be treated as being independent of child max.